systemtest/src/test/java/io/strimzi/systemtest/operators/topic/ThrottlingQuotaST.java

```diff
@@ -0,0 +1,417 @@
+        PodUtils.waitUntilMessageIsInPodLogs(
+                createPod.getMetadata().getName(),
+                "org.apache.kafka.common.errors.ThrottlingQuotaExceededException: The throttling quota has been exceeded.",
+                Duration.ofMinutes(5).toMillis()
+        );
+        LOGGER.debug("=================\n" + kubeClient().logs(createPod.getMetadata().getName()));
+        JobUtils.deleteJobWithWait(namespaceName, createAdminName);
+
+        // Delete created topics (as they were created in random order, we have to use KafkaTopic instead of this client) (really?)
+        KafkaTopicList ktl = KafkaTopicResource.getAllKafkaTopicsWithPrefix(namespaceName, topicNamePrefix);
+        for (KafkaTopic kt : ktl.getItems()) {
+            cmdKubeClient().deleteByName(KafkaTopic.RESOURCE_SINGULAR, kt.getMetadata().getName());
+        }
+    }
+
+    @ParallelNamespaceTest
+    @Tag(INTERNAL_CLIENTS_USED)
+    void testThrottlingQuotasDeleteTopic(ExtensionContext extensionContext) {
+        final String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
+        final String namespaceName = StUtils.getNamespaceBasedOnRbac(NAMESPACE, extensionContext);
+        int topicsCountOverQuota = 500;
+        String createAdminName = "create-admin-client";
+        String deleteAdminName = "delete-admin-client";
+        String topicNamePrefix = classTopicPrefix + "-delete";
+
+        setupKafkaInNamespace(extensionContext, clusterName, namespaceName);
+
+        // Create many topics
+        KafkaBasicExampleAdminClient createAdminClientJob;
+        Pod createPod;
+        int offset = 100;
+        for (int i = 0, j = 0; i < topicsCountOverQuota; i = i + offset, j++) {
+            LOGGER.info("Executing {}/{} iteration.", j + 1, topicsCountOverQuota / offset);
+            createAdminClientJob = new KafkaBasicExampleAdminClient.Builder()
+                .withAdminName(createAdminName)
+                .withBootstrapAddress(KafkaResources.plainBootstrapAddress(clusterName))
+                .withTopicName(topicNamePrefix)
+                .withTopicCount(100)
+                .withTopicOffset(i)
+                .withNamespaceName(namespaceName)
+                .withTopicOperation("create")
+                .withAdditionalConfig(getAdminClientConfig(namespaceName))
+                .build();
+            resourceManager.createResource(extensionContext, true, createAdminClientJob.adminStrimzi().build());
+            ClientUtils.waitForClientSuccess(createAdminName, namespaceName, 10);
+            createPod = kubeClient().listPods("job-name", createAdminName).get(0);
+            PodUtils.waitUntilMessageIsInPodLogs(createPod.getMetadata().getName(), "All topics created");
+            LOGGER.debug("=================\n" + kubeClient().logs(createPod.getMetadata().getName()));
+            JobUtils.deleteJobWithWait(namespaceName, createAdminName);
+        }
```

Comment:
I guess this is not enough readable (from MPOV). Also, wouldn't it be better to load all parameters, which are same, to the `createAdminClientJob` before the loop and then just do `toBuilder()` and change the topic offset, which seems to be only thing that is different?